pkg/apis/operator/v1alpha1/tektonchain_types.go

```diff
@@ -62,8 +62,15 @@ type TektonChainSpec struct {
 	Config Config `json:"config,omitempty"`
 }
 
-// Chain defines the field to provide chain configuration
 type Chain struct {
+	ChainProperties `json:",inline"`
+	// The params to customize different components of Pipelines
+	// +optional
+	Params []Param `json:"params,omitempty"`
```

Comment:
If possible add a test to verify `params` field

Reply:
Since there are no restricted values and need to access the deployment to see the env, hence I think it is fine if we not add a test for this